core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -114,6 +118,17 @@ public UnderFileSystemConfiguration setShared(boolean shared) {
   public UnderFileSystemConfiguration createMountSpecificConf(Map<String, String> mountConf) {
     UnderFileSystemConfiguration ufsConf = new UnderFileSystemConfiguration(mProperties.copy());
     ufsConf.mProperties.merge(mountConf, Source.MOUNT_OPTION);
+    String ufsConfigValue = ufsConf.get(PropertyKey.UNDERFS_HDFS_CONFIGURATION);
```

Comment:
Can you update the property key for `UNDERFS_HDFS_CONFIGURATION` to describe this new feature?

Reply:
Done.